.vib/codeigniter/vib-publish.json

```diff
@@ -3,7 +3,8 @@
     "resources": {
       "url": "{VIB_ENV_CONTAINER_URL}",
       "path": "{VIB_ENV_PATH}"
-    }
+    },
+    "runtime_parameters": "Y29tbWFuZDogWyJ0YWlsIiwgIi1mIiwgIi9kZXYvbnVsbCJd"
```

Comment:
We could take the opportunity to enable `runtime_parameters_file` here.

Reply:
Wdyt @FraPazGal?

Reply:
As of now, the runtime params for containers only accept a single parameter, which is the `command` one. This is used to override the container's default one. Unlike `bitnami/charts`, here we do not have custom parameters that actively affect the tests and are prone to changes when the chart is updated.

Maybe in the future, when in-depth test suites are created for some of these containers, we will use more complex runtime parameters, and having them in a `yaml` file adds some values. Until then, I think using the encoded string is enough.